### PR TITLE
updated palindrome_number and its tests

### PR DESCRIPTION
**Pull request by ewatki**

### CODE REVIEW

File: tests/palindrome_number_test.py [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+    n = 99
+
+    # Act
+    answer = palindrome_number(99)
+
+    # Assert
+    assert answer == True
+
+
+def test_returns_true_for_151():
```
Comment (angela100743) [inline tests/palindrome_number_test.py:15]: Nice job!

File: tests/palindrome_number_test.py [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
+
+def test_returns_true_for_99():
+    # Arrange
+    n = 99
+
+    # Act
+    answer = palindrome_number(99)
+
+    # Assert
+    assert answer == True
```
Comment (angela100743) [inline tests/palindrome_number_test.py:12]: This looks pretty good. Here's another try:
'''python
assert answer == False
pass'''

**Review by angela100743 — COMMENTED** [review]
This looks phenomenal

File: tests/palindrome_number_test.py (hunk 1 above)
Comment (MotifNoticer) [reply tests/palindrome_number_test.py:15]: All of these test are passing. Nice work!

**Review by MotifNoticer — APPROVED** [review]

**Pull request merged**